cg/cli/workflow/rnafusion/base.py

```diff
@@ -60,23 +48,17 @@ def rnafusion(context: click.Context) -> None:
 @OPTION_STRANDEDNESS
 @DRY_RUN
 @click.pass_obj
-def config_case(
-    context: CGConfig,
-    case_id: str,
-    strandedness: str,
-    dry_run: bool,
-) -> None:
+def config_case(context: CGConfig, case_id: str, strandedness: str, dry_run: bool) -> None:
     """Create samplesheet file for RNAFUSION analysis for a given CASE_ID."""
-
-    analysis_api: AnalysisAPI = context.meta_apis["analysis_api"]
-
+    analysis_api: RnafusionAnalysisAPI = context.meta_apis["analysis_api"]
```

Comment:
Do we have constants for "analysis_api" ? This string is repeated below as well. If there is none we should make one in cg.constants. 

It would look somehting like that in the end

```suggestion
    analysis_api: RnafusionAnalysisAPI = context.meta_apis[ConstantClassName.ANALYSIS_API]
```